src/syzygy/syzygy.h

```diff
@@ -20,6 +20,15 @@ enum WdlResult {
 
 namespace {
 
+uint8_t s_dtz {};
+WdlResult s_wdl { WdlResultTableNotActive };
+
+inline void resetResults()
```

Comment:
Reset is probably enough 
```suggestion
inline void reset()
```